modules/EPMC.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+
+import argparse
+import sys
+from pyspark import *
+from pyspark.sql import *
+from pyspark.sql.types import *
+from pyspark.sql.functions import *
+from pyspark.sql.window import Window
+import logging
+
+
+# example run local, make sure you dont pass many parquet files as input to not run out of mem
+# also, double check you put the ram size properly in the local settings in the SparkConf map
+# python modules/EPMC.py --local \
+#        --cooccurrenceFile /home/mkarmona/src/opentargets/data/platform/epmc/epmc-cooccurrences \
+#        --outputFile test_out
+#        --logFile epmc_parser.log
```

Comment:
```suggestion
# Example for a local run. Make sure you don't pass many parquet files as input to avoid running out of memory.
# Also, double check you put the RAM size properly in the local settings in the SparkConf map.
# python modules/EPMC.py --local \
#        --cooccurrenceFile /home/mkarmona/src/opentargets/data/platform/epmc/epmc-cooccurrences \
#        --outputFile test_out
#        --logFile epmc_parser.log
```

Reply:
Also, should this information perhaps be moved to the documentation (README)?

Reply:
I don't think so. These are mostly remarks for myself, I should have removed it. Regarding documentation in general, I would not go excessive with it. The readme of the repository should provide enough information to set up the environment and run the pipeline. The individual script should provide enough help on how it should be called (`--help`), that's probably the easiest thing to keep up to date. Also a good logging is informative. My biggest concern about documentation is that there's a big risk that the documentation, wherever it lives, diverge from the actual behaviour. 

Other than there shouldn't be really a need for anything else.

Reply:
Agreed. Especially about bundling the documentation with the codebase (in the form of parser help/comments/docstrings), this is indeed a great way to ensure it doesn't get out of sync with reality